Notebook for UC Migration Workflow Scoping

prashsub opened the pull request:
TLC-820 How to scope a UC Workflow Migration using System Tables

matthewmoorcroft commented:
@prashsub can you have SME comment here that they approve the code?

dmoore247 commented:
I'm the blog reviewer...
1. The notebook should not contain a default workspace_id value that might link back to databricks or a customer...
2. use `:param` instead of $params. 
3. Command 5 text cell promises totals, command 6 sql does not total.
4. personal catalog is named in example: `prashanth_subrahmanyam_catalog`
5. Command 19 could be an interesting visual, but notebook doesn't have any visualizations. Have you considered putting this into a dashboard?

Changes are required before approval.

prashsub commented:
Hi @dmoore247,

Thank you for the feedback.
1. I removed the default workspace_id in the notebook and converted to .py format.
2. I couldn't replace $params with :param as the new notation is still not supported in DDL statements like Creating Views.
3. Revised the text in Cell 5 to say list instead of count.
4. Removed reference to personal catalog and replaced with main instead.
5. I added an accompanying dashboard into the repo for visualizations.

matthewmoorcroft reviewed (approved):
Looks all good
